webview-ui/src/CreateFleet/helpers/state.ts

```diff
@@ -0,0 +1,91 @@
+import {
```

Comment:
could you add comment at the top of this file to explain what state this file defines and encodes, where it's used, etc?

Reply:
this state.ts file is required for communication between webview and extension. webview is for user experience (UI components) and actual extension handles everything with the vscode.  this file is for state management of create fleet flow I believe.